RiotSwiftUI/Modules/Room/VoiceBroadcastRecorder/Service/MatrixSDK/VoiceBroadcastRecorderService.swift

```diff
@@ -176,9 +186,11 @@ class VoiceBroadcastRecorderService: VoiceBroadcastRecorderServiceProtocol {
     }
     
     /// Send chunk file to the server.
-    private func sendChunkFile(at url: URL, sequence: Int) {
-        guard let voiceBroadcastService = voiceBroadcastService else {
+    private func sendChunkFile(at url: URL, sequence: Int, completion: @escaping () -> Void = {}) {
```

Comment:
I think it's better to have this completion block like this :
`completion: @escaping (() -> Void)? = nil`